transforms/replace-prop-value/README.md

```diff
@@ -54,6 +54,10 @@ If there is a pattern that you use that is not transformed, please file a featur
 
 `npx carbon-codemod replace-prop-value <target> <component-import-path> <prop> <old-value> <new-value>`
 
+Note: To use a string as `old-value` or `new-value`, format it as `\"string\"` on the command line, i.e.: 
```

Comment:
I think we need to update this, because the example below doesn't need to be escaped. It's only if you want to use a numeric value as a string that you need to quote it.